cds/modules/deposit/static/js/cds_deposit/avc/components/cdsForm.js

```diff
@@ -212,28 +212,35 @@ function cdsFormCtrl($scope, $http, $q, schemaFormDecorators) {
     // Response handler
     function (data, query) {
       var userInput = authorFromUser(query);
-      var suggestions = data.data.map(function (author) {
+      var suggestions = data.data.map(function (hit) {
         var valueObj = {};
-        valueObj.name = author.sn + ", " + author.givenName;
-        if (author.cernInstituteName) {
-          valueObj.affiliations = [author.cernInstituteName];
+        // Data structure depends on the type of the object
+        if (hit.cernAccountType) {
```

Comment:
I chose `cernAccountType` but here is the difference between person vs e-group in case anyone has a better suggestion (it's the same endpoint)
![image](https://user-images.githubusercontent.com/15194802/177334258-e8100bbc-6a89-4ebe-ae12-e6b81c686b63.png)